deploy/helm/flightctl/templates/api/flightctl-api-config.yaml

```diff
@@ -78,6 +78,7 @@ data:
         k8s:
             apiUrl: {{ .Values.global.auth.k8s.apiUrl }}
             rbacNs: {{ default .Release.Namespace .Values.global.auth.k8s.rbacNs }}
+            roleSuffix: {{ .Release.Name }}
```

Comment:
we should use `.Release.Namespace` everywhere.
You can deploy 1 instance of fctl into each ns. But all instances can have the same release name